### DISCUSSION TITLE
my Monika didn't celebrate Halloween.

### DISCUSSION THREAD

**Discussion by howltrek** [Monika-After-Story/MonikaModDev#8238] [QA - tech support + general] [answered]
she did nothing for the date.

Also for many days no new dialogues appear either, I don't know what to do

**Comment by jmwall24** [comment]
what time is it currently?

**Comment by howltrek** [comment]
> what time is it currently?

0:30

**Comment by jmwall24** [comment, marked as answer]
You're fine then, Halloween event doesn't start until 3am if you start a new sesh after 3, or 6am if you have left it open from midnight onwards.
